src/Components/BreathingRoom/BreathingRoom.scss

```diff
@@ -0,0 +1,44 @@
+.BreathingRoom{
```

Comment:
I wonder if we could use the `styled-component` stuff with animations?? I'm sure we don't need to, and I'm not sure if there would be a use case for it, but. I have to wonder. Maybe next time we meet we can discuss possible use cases for that?